docs/cpp_docs.rst

```diff
@@ -7,15 +7,33 @@ Documenting C++ Code
    This is a preview documentation format specification.
    Software documentation should currently be written in the format described at https://confluence.lsstcorp.org/display/LDMDG/Documentation+Standards
 
```

Comment:
At this stage I'd be tempted to just drop the warning :) Do we know what relevant material is left on that confluence page?